cinema/views.py

```diff
@@ -101,8 +104,24 @@ def get_serializer_class(self):
         if self.action == "retrieve":
             return MovieDetailSerializer
 
+        if self.action == "upload_image":
+            return MovieImageSerializer
```

Comment:
Ensure that `MovieImageSerializer` is appropriate for handling image uploads. It should be able to process the image data correctly and update the `Movie` instance.